### ISSUE TITLE
xdl 优化器使用

### ISSUE THREAD

**Issue by songziqin** [alibaba/x-deeplearning#281]
我这边的特征有两种，一种是稀疏特征，一种是稠密特征，我想问下，如果我的稀疏特征想使用FTRL优化器，稠密特征使用ADAM 优化器，是不是现在还不支持，需要我这边再开发一个优化器来：
模仿class Ftrl(Optimizer):
    def __init__(self, 
                 learning_rate, 
                 learning_rate_power=-0.5,
                 initial_accumulator_value=0.1,
                 l1_regularization_strength=0.0,
                 l2_regularization_strength=0.0):
        """construct a ftrl optimizer
           Args:
             learning_rate: a float value indicate learning rate
             learning_rate_power: a float value, must be less or equal to zero
             initial_accumulator_value: the starting value for accumulators
             l1_regularization_strength: a float value, must be greater than or equal to zero
             l2_regularization_strength: a float value, must be greater than or equal to zero
        """
        super(Ftrl, self).__init__()
        self._lr = learning_rate
        self._lr_power = learning_rate_power
        self._init_acc = initial_accumulator_value
        self._l1_reg = l1_regularization_strength
        self._l2_reg = l2_regularization_strength
    
    def dense_update(self, var, grad):
        return xdl.ps_dense_apply_ftrl_op(
            learning_rate = self._lr,
            learning_rate_power = self._lr_power,
            initial_accumulator_value = self._init_acc,
            l1_reg = self._l1_reg,
            l2_reg = self._l2_reg,
            grad = grad,
            var_name = var.name,
            var_type = var.vtype)

    def sparse_update(self, var, grad, indices):
        return xdl.ps_sparse_apply_ftrl_op(
            learning_rate = self._lr,
            learning_rate_power = self._lr_power,
            initial_accumulator_value = self._init_acc,
            l1_reg = self._l1_reg,
            l2_reg = self._l2_reg,
            grad = grad,
            indices = indices,
            var_name = var.name,
            var_type = var.vtype)
只是dense_update、sparse_update使用不同的接口呢。

**Comment by songyue1104** [comment]
optimizer的optimize接口有个var_list参数，可以指定需要优化的variable
你可以创建两个optimizer，传入不同的variable列表
